Packs/CommonScripts/Scripts/DockerHardeningCheck/DockerHardeningCheck.py

```diff
@@ -150,9 +159,50 @@ def check_cpus(num_cpus: int) -> str:
     return ""
 
 
+def get_default_gateway():
+    res = subprocess.check_output(["ip", "route", "list"], text=True, stderr=subprocess.STDOUT)
+    LOG(f'result of ip route list: {res}')
+    line1 = res.splitlines()[0]
+    if not line1.startswith('default via'):
+        raise ValueError(f'Excpected "ip route list" to start with "default via" but not found. Got: [{line1}]')
+    return line1.split()[2]
+
+
+def check_network(network_check: str) -> str:
+    """
+    Check that Cloud provider metadata service is not exposed and that access to localhost is not available.
+    """
+    return_res = ""
+    if network_check in ("all", "cloud_metadata"):
+        LOG('Check cloud metadata server access...')
```

Comment:
If we're logging, then we can write why we got here ("all" or "cloud_metadata").
```suggestion
        LOG('Check cloud metadata server access... {network_check=}')
```